### PR TITLE
DM-32960: Set up as SPIE paper and write abstract

### PR DESCRIPTION
**Pull request by leannep**

### CODE REVIEW

**Comment by leannep** [comment]
Note there is a 250 word limit on the abstract

**Review by womullan — APPROVED** [review]
ok thanks

**Comment by leannep** [comment]
SPIE also requests a 100 work summary in addition to the 250 word abstract.I propose to following: 

Once operational, up to 20TB of raw imaging data will be collected by Rubin Observatory's Legacy Survey of Space and Time (LSST) per night and processed by the LSST Science Pipelines to produce science-ready data products – processed images, catalogs and alerts.  To ensure that these data products can enable transformative science with LSST,  stringent requirements have been placed on their quality and scientific fidelity. In this paper we introduce faro,  a framework to efficiently quantify the scientific performance of the LSST data products for data units of varying granularity, ranging from single-detector to full-survey summary statistics.

File: abstract.tex [hunk 1]
```diff
@@ -0,0 +1,14 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize all areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
```
Comment (bechtol) [inline abstract.tex]: To be fair, LSST might not revolutionize _all_ areas of astronomy. For example, other facilities are much more effective for exoplanet research. Maybe think about an alternative phrasing.

Comment (leannep) [reply abstract.tex]: many areas of astronomy?

File: abstract.tex [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
+to generate science-ready data products -- processed images, catalogs and alerts. 
+The LSST Science Requirements Document (SRD) defines Key Performance Metrics (KPMs), scalar quantities that measure the quality and scientific fidelity of the LSST data products, together with specifications that represent the capability or accuracy required by the system to achieve the scientific goals.
```
Comment (bechtol) [inline abstract.tex]: I wonder if we should be so heavily emphasizing the KPMs. On one level, this is a very simple story that we can tell, and the SRD is an accessible user-facing document (much more so than the LSR, OSS, and DMSR). On the other hand, we are envisioning a system that computes a much wider range of metrics than the KPMs. For the purpose of this abstract though, keeping things simple may be advantageous.

File: abstract.tex [hunk 3]
```diff
@@ -0,0 +1,14 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize all areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
+LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
+to generate science-ready data products -- processed images, catalogs and alerts. 
+The LSST Science Requirements Document (SRD) defines Key Performance Metrics (KPMs), scalar quantities that measure the quality and scientific fidelity of the LSST data products, together with specifications that represent the capability or accuracy required by the system to achieve the scientific goals. 
+KPMs are defined for a wide range of quantities such as image quality and depth, astrometric and photometric performance, and object recovery completeness. 
+By measuring and tracking KPMs we are able to evaluate trends in algorithmic performance, or compare the performance of one algorithm against another, 
+By comparing measured KPMs to specifications we can verify that the LSST data products will meet requirements.
```
Comment (edennihy) [inline abstract.tex]: minor typo - change comma to period

Comment (leannep) [reply abstract.tex]: 👍🏻

File: abstract.tex [hunk 4]
```diff
@@ -0,0 +1,14 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize all areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
+LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines
```
Comment (edennihy) [inline abstract.tex]: I think this should be "The LSST will produce" following the use guide: https://project.lsst.org/documents/name-use-guide

**Review by edennihy — COMMENTED** [review]
A few minor comments added in-line to the abstract file.

File: abstract.tex (hunk 4 above)
Comment (leannep) [reply abstract.tex]: Yes, updated

File: abstract.tex (hunk 3 above)
Comment (leannep) [reply abstract.tex]: fixed

File: abstract.tex (hunk 2 above)
Comment (jeffcarlin) [reply abstract.tex]: I agree with Keith - the KPMs are a (very important!) subset of the variety of metrics we can (and will) calculate. (But it's also fine to leave the abstract as it is, and address this point in the text later...)

File: abstract.tex [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+to generate science-ready data products -- processed images, catalogs and alerts. 
+The LSST Science Requirements Document (SRD) defines Key Performance Metrics (KPMs), scalar quantities that measure the quality and scientific fidelity of the LSST data products, together with specifications that represent the capability or accuracy required by the system to achieve the scientific goals. 
+KPMs are defined for a wide range of quantities such as image quality and depth, astrometric and photometric performance, and object recovery completeness. 
+By measuring and tracking KPMs we are able to evaluate trends in algorithmic performance, or compare the performance of one algorithm against another.
+By comparing measured KPMs to specifications we can verify that the LSST data products will meet requirements. 
+In this paper we present \textit{faro}, a framework to efficiently quantify the scientific performance of the LSST data products for data units of varying granularity, ranging from single-detector to full-survey summary statistics, and to persist the results as scalar metric values alongside the input data products.
```
Comment (jeffcarlin) [inline abstract.tex]: Should we use \texttt{faro}? I think that's how astro journals would do it, but not sure if SPIE is different.

File: abstract.tex [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+We present results using \textit{faro} to characterize the performance of the data products produced on simulated and precursor data sets, and 
+ discuss plans to use faro to verify the performance of the LSST commissioning data products.
+ 
+\noindent \textbf{100 word summary}
+
+\noindent Once operational, up to 20TB of raw imaging data will be collected by Rubin Observatory's Legacy Survey of Space and Time (LSST) per night and processed by the LSST Science Pipelines to produce science-ready data products ? processed images, catalogs and alerts.  To ensure that these data products can enable transformative science with LSST,  stringent requirements have been placed on their quality and scientific fidelity. In this paper we introduce faro,  a framework to efficiently quantify the scientific performance of the LSST data products for data units of varying granularity, ranging from single-detector to full-survey summary statistics.
```
Comment (jeffcarlin) [inline abstract.tex]: Note: "faro" in the last sentence isn't italicized (or typewriter font).

**Review by jeffcarlin — APPROVED** [review]
Looks good - thanks, Leanne!

File: abstract.tex (hunk 6 above)
Comment (leannep) [reply abstract.tex]: fixed

File: abstract.tex (hunk 5 above)
Comment (leannep) [reply abstract.tex]: So I did a quick search, \texttt is used maily for code. faro is the name of a code package so i think you are right.  The SPIE template uses {\ttfamily faro}, which renders basically the same as \texttt, so I'll use the SPIE example.

File: abstract.tex (hunk 2 above)
Comment (leannep) [reply abstract.tex]: I am trying to rewrite this in a more general manner.

File: abstract.tex (hunk 5 above)
Comment (leannep) [reply abstract.tex]: Updated - please check

File: abstract.tex (hunk 2 above)
Comment (leannep) [reply abstract.tex]: Let's keep specific discussion on KPMs for the main text and be more general in the abstract - new version committed. Please re-review

File: abstract.tex [hunk 7]
```diff
@@ -0,0 +1,18 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize many areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
+The LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
+to generate science-ready data products -- processed images, catalogs and alerts. 
+To ensure that these data products can enable transformative science with LSST, stringent requirements, have been placed on their quality and scientific fidelity, for example on image quality and depth, astrometric and photometric performance, and object recovery completeness.
```
Comment (jeffcarlin) [inline abstract.tex]: "can enable" --> "enable"?

Also, no comma needed after "stringent requirements"

Comment (leannep) [reply abstract.tex]: Fixed

File: abstract.tex [hunk 8]
```diff
@@ -0,0 +1,18 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize many areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
```
Comment (bechtol) [inline abstract.tex]: "will advance nearly every area of astronomy"

**Review by bechtol — APPROVED** [review]
Thanks for these revisions. I think this version of the abstract captures the main points well.

File: abstract.tex (hunk 8 above)
Comment (leannep) [reply abstract.tex]: nice - updated

**Pull request merged**